.gitignore

```diff
@@ -18,3 +18,5 @@ coverage
 !.yarn/releases
 !.yarn/sdks
 !.yarn/versions
+
+.vscode/
```

Comment:
Can you put this in your a local only file to ignore like `.git/info/exclude` or a global gitignore https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Sure thing